x-pack/auditbeat/module/system/package/package_homebrew_test.go

```diff
@@ -0,0 +1,103 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+// +build !windows
+
+package pkg
+
+import (
+	"os"
+	"runtime"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/elastic/beats/auditbeat/core"
+	abtest "github.com/elastic/beats/auditbeat/testing"
+	"github.com/elastic/beats/libbeat/beat"
+	mbtest "github.com/elastic/beats/metricbeat/mb/testing"
+)
+
+func TestHomebrew(t *testing.T) {
+	defer abtest.SetupDataDir(t)()
+
+	oldPath := homebrewCellarPath
+	defer func() {
+		homebrewCellarPath = oldPath
+	}()
+	homebrewCellarPath = "../../../tests/files/homebrew/"
```

Comment:
The conventional directory for this in Go would be `testdata` within this package’s directory. `testdata` is ignored by Go (I think this is documented in `go help test`).

Reply:
Hm, so `tests/files` already contains some other test files as well. Should they all move to a `testdata`?

Reply:
If they are used by the Go tests then I would move them into a testdata dir next to the tests that use them. It's not a hard rule, but it's a pretty common practice with this sort of data as also with golden-file tests.

Reply:
Ok, makes sense. For now - in this PR - I've moved the new `homebrew` directory. We should move the others later.